book_source/source/module-5-classes-and-objects/lesson-14-more-classes-and-modules/private-fields.md

```diff
@@ -5,6 +5,7 @@
     <iframe src="https://www.loom.com/embed/de5c457d29854cad9093650e160fc9da?sharedAppSource=personal_library" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
 </div>
 
+---
```

Comment:
Same things as the last PR. There is a blocquote on this page (that I can't comment on directly) that ate the next paragraph.